omod/src/main/webapp/fragments/patientdashboard/activeDrugOrders.gsp

```diff
@@ -17,7 +17,7 @@
         <i class="icon-medicine"></i>
         <h3>${ ui.message("orderentryui.patientdashboard.activeDrugOrders").toUpperCase() }</h3>
         <% if (context.hasPrivilege("App: orderentryui.drugOrders")) { %>
-            <a href="${ ui.pageLink("orderentryui", "drugOrders", [patient: patient.id, returnUrl: ui.thisUrl()]) }">
+        <a href="${ "../../owa/openmrs-owa-orderentry/index.html#/?patient="+patient.uuid }">
```

Comment:
What is the hash used for in this case?
The same goes for the last slash.

Reply:
The `#/` is included to allow access to other routes that may be needed in the application. If we would not need to access other routes, for instance, `/drugs` or similar routes, then the `#/` is not needed and can be removed.

Reply:
So what happens if we do not have #/ in the above?

Reply:
The following error message is returned - `Object with given uuid doesn't exist [null]`

Reply:
That error happens with what exact url?

Reply:
Yes.
However, there is also the possibility of implementing the logic in the OWA we are linking to such that the `#/` would not be needed here, so it can be safely removed.

Reply:
Can you remove the #/ and the attach the compiled app on the ticket for me to test and reproduce?

Reply:
Absolutely.

Reply:
Done.
Here is the link to the compiled omod - https://issues.openmrs.org/projects/OEUI/issues/OEUI-40

Reply:
And what are the steps of reproducing it?

Reply:
Steps to reproduce the error:
* Upload the compiled omod file by navigating to `System Administration >> Advanced Administration >> Open Web Apps Module (Manage Apps) >> Add On Manager`.
* Upload the omod file
* Navigate to the home screen >> Find patient Record >> PRESCRIBED MEDICATION
* Click on the  arrow icon to navigate to the OWA